### DISCUSSION TITLE
to com probelam no meu node js

### DISCUSSION THREAD

**Discussion by lucccansousa** [balta-io/forum#341] [Tira Dúvidas] [answered]
![image](https://github.com/balta-io/forum/assets/131711446/22adaaaf-bfc9-42fa-8ad2-eae78a09ce9c)
me ajudaaa

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @lucccansousa, você precisa explicar melhor as coisas que precisa, senão fica difícil te ajudar.
Já é a terceira vez que você cria um tópico só com uma imagem e pedindo ajuda.

Se você não se dispões a explicar o que precisa, por que acredita que deve me dispor a te ajudar?
Vamos fazer o seguinte, você detalha o que precisa e te ajudo com todos os detalhes.

**Reply by lucccansousa** [reply]
então toda vez que eu entro no node  dar esse erro, ja tentei de tudo apaguei o node e instalei denovo esse erro da imagem

**Reply by andrebaltieri** [reply]
Bom dia, @lucccansousa, como vai?

O Node é só um executável, não tem necessidade de acessar ele, você pode chamar ele via terminal, como por exemplo:
`node meuApp.js`

**Reply by lucccansousa** [reply]
eu preciso resolver esse erro pq formatei meu pc e apareceu isso
vc sabe que erro e essee??

**Reply by andrebaltieri** [reply]
Bom dia, @lucccansousa, como vai?

Quais passos você faz até chegar neste erro? Qual aula, de qual curso isto ocorre?

**Reply by codetrindade** [reply]
É um bug conhecido que foi corrigido.

https://github.com/nodejs/node/issues/3610
https://github.com/nodejs/node/pull/3630

Tente atualizar para uma versão mais recente.
